### ISSUE TITLE
the imsize

### ISSUE THREAD

**Issue by easezyc** [heykeetae/Self-Attention-GAN#5]
In your code, there is the imsize, howerver in the paper there is no imsize. The  int(in_dim / 8)*(imsize ** 2) means the channel of the output(the \hat{C} right?).
self.f_ = nn.Conv2d(in_dim, int(in_dim / 8)*(imsize ** 2), 1)
self.g_ = nn.Conv2d(in_dim, int(in_dim / 8)*(imsize ** 2), 1)

**Comment by heykeetae** [comment]
Yes imsize refers to image size, 64 in this implementation. To be compared with the paper, the feature nunber N is the same as imsize^2. Imsize is also interchangeably used as f_size in  the Self_Attn module, but Let me make notations clear soon.
